packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -149,7 +149,13 @@ export interface IFluidDataStoreContextProps {
 	readonly storage: IDocumentStorageService;
 	readonly scope: FluidObject;
 	readonly createSummarizerNodeFn: CreateChildSummarizerNodeFn;
-	readonly pkg?: Readonly<string[]>;
+	/**
+	 * See {@link FluidDataStoreContext.pkg}.
+	 */
+	readonly pkg?: PackagePath;
+	/**
+	 * See {@link FluidDataStoreContext.loadingGroupId}.
+	 */
```

Comment:
Should these just use `@inheritDoc`?

Reply:
For APIs which are not package exported, I like links which I can click in intelisense instead of inheritDoc which is not handled in any useful way by any tool which is used on this comment.